jmxtrans-cluster/src/main/java/com/googlecode/jmxtrans/cluster/ClusterConfigChangeListener.java

```diff
@@ -0,0 +1,11 @@
+package com.googlecode.jmxtrans.cluster;
+
+/**
+ * Created by kulcsart on 5/6/2016.
+ */
+@Deprecated
```

Comment:
I suspect you will remove this class (and other deprecated classes) before the final version...